src/Be.Vlaanderen.Basisregisters.Aws.Lambda/FunctionBase.cs

```diff
@@ -83,7 +83,7 @@ public async Task Handler(JObject @event, ILambdaContext context)
                 var pingMessage = @event.ToObject<PingEvent>();
                 if (pingMessage is not null)
                 {
-                    context.Logger.LogInformation($"Ping: {pingMessage} received.");
+                    context.Logger.Log($"Ping: {pingMessage} received.");
```

Comment:
Here you have replaced context.Logger.LogInformation by just Log. Below on line 108 you've replaced logger.LogDebug by logger.Log. Can't we pass a long the log level?